packages/format-library/src/bold/index.js

```diff
@@ -18,6 +18,7 @@ export const bold = {
 	title,
 	tagName: 'strong',
 	className: null,
+	__unstableEssential: true,
```

Comment:
Core should no longer ship public APIs with `__unstable|experimental` prefixes. We should either make it private or stable.